nixos/modules/security/pam.nix

```diff
@@ -470,90 +578,116 @@ let
       setLoginUid = mkDefault cfg.startSession;
       limits = mkDefault config.security.pam.loginLimits;
 
+      text = let
+        ensureUniqueOrder = type: rules:
+          let
+            headRules = if rules == [] then [] else [ (head rules) ];
+            tailRules = if rules == [] then [] else tail rules;
```

Comment:
```suggestion
            headRules = take 1 rules;
            tailRules = drop 1 rules;
```

Reply:
Updated in 077cdcc7e9a9ed406860e97ce3590fba15aed18b.